site/src/pages/UserSettingsPage/WorkspaceProxyPage/WorkspaceProxyRow.tsx

```diff
@@ -0,0 +1,81 @@
+import { Region } from "api/typesGenerated"
+import { AvatarData } from "components/AvatarData/AvatarData"
+import { Avatar } from "components/Avatar/Avatar"
+import { useClickableTableRow } from "hooks/useClickableTableRow"
+import TableCell from "@material-ui/core/TableCell"
+import TableRow from "@material-ui/core/TableRow"
+import { FC } from "react"
+import {
+  HealthyBadge,
+  NotHealthyBadge,
+} from "components/DeploySettingsLayout/Badges"
+import { makeStyles } from "@material-ui/core/styles"
+import { combineClasses } from "utils/combineClasses"
+
+export const ProxyRow: FC<{
+  proxy: Region
+  onSelectRegion: (proxy: Region) => void
+  preferred: boolean
+}> = ({ proxy, onSelectRegion, preferred }) => {
+  const styles = useStyles()
+
+  const clickable = useClickableTableRow(() => {
+    onSelectRegion(proxy)
+  })
+
+  const classes = [clickable.className]
+
+  if (preferred) {
+    classes.push(styles.preferredrow)
+  }
+
+  return (
+    <TableRow
+      key={proxy.name}
+      data-testid={`${proxy.name}`}
+      {...clickable}
+      // Make sure to include our classname here.
+      className={combineClasses(classes)}
```

Comment:
Here's a cleaner way to use conditional classes:
```suggestion
      className={combineClasses({
          [clickable.className]: true,
          [styles.preferredrow]: preferred,
        })}
```

Reply:
TIL